cdk/typescript/cdk.json

```diff
@@ -0,0 +1,64 @@
+{
+  "app": "npx ts-node --prefer-ts-exts bin/the-lambda-power-tuner.ts",
+  "watch": {
+    "include": [
+      "**"
+    ],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "**/*.d.ts",
+      "**/*.js",
+      "tsconfig.json",
+      "package*.json",
+      "yarn.lock",
+      "node_modules",
+      "test"
+    ]
+  },
+  "context": {
```

Comment:
What is this for? Do we need all this EC2/ECS/RDS stuff?

Reply:
> What is this for? Do we need all this EC2/ECS/RDS stuff?

This file points NPM to execute the "bin" file whenever a CDK command is executed.

Reply:
> > What is this for? Do we need all this EC2/ECS/RDS stuff?
> 
> This file points NPM to execute the "bin" file whenever a CDK command is executed.

I understand, but why do we need all the EC2/ECS/RDS stuff? Is it required for this CDK project?